config/migrations/2023/lpdc/20230512160000-add-concept-display-configurations.sparql

```diff
@@ -0,0 +1,36 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX cpsv: <http://purl.org/vocab/cpsv#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX lpdcExt: <https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX pav: <http://purl.org/pav/>
+
+INSERT {
+  GRAPH ?graph {
+    ?concept lpdcExt:hasConceptDisplayConfiguration ?configUri .
+    ?configUri a lpdcExt:ConceptDisplayConfiguration ;
+      mu:uuid ?configId ;
+      lpdcExt:conceptIsNew true ;
+      lpdcExt:conceptInstantiated false ;
+      dct:relation ?eenheid .
+  }
+}
+WHERE {
+  VALUES ?eenheidClassificatie {
+    <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000001> # gemeente
```

Comment:
@Windvis same remark, double check the list thanks!

Reply:
We also need to add "Provincies" to the list here. Do I create a separate migration for each type to limit the chance of timeouts?

Reply:
yeah how many provincies do we have these days again? :-)